src/ArraySerializable.php

```diff
@@ -68,6 +68,10 @@ public function hydrate(array $data, $object)
         }
 
         if (is_callable([$object, 'exchangeArray'])) {
+            if (is_callable([$object, 'getArrayCopy'])) {
+                $original = $object->getArrayCopy($object);
+                $replacement = \Zend\Stdlib\ArrayUtils::merge($original, $replacement);
```

Comment:
Import the class.

(I'll take care of this on merge, however, as it's the only thing sticking out as needing fixing at this time.)